tests/informatics/test_predictors.py

```diff
@@ -1,7 +1,7 @@
 """Tests for citrine.informatics.predictors."""
 import uuid
-import mock
 import pytest
+import mock
```

Comment:
?

Also, I think we've been trying to get rid of usages of the mock library in favor of pytest's mocking.

Reply:
Probably just shuffling around tests and imports getting removed and reintroduced